ui/StatusQ/src/leftjoinmodel.cpp

```diff
@@ -74,7 +74,7 @@ void LeftJoinModel::initialize(bool reset)
     if (!namesIntersection.isEmpty()) {
         qWarning().nospace() << "Source models contain conflicting model names: "
                              << QList(namesIntersection.cbegin(),
-                                      namesIntersection.cend())
+                                      namesIntersection.cend()).join()
```

Comment:
I think you should be able to just stream the `namesIntersection` set to qWarning()

Reply:
Sure, not a big deal